src/constants.js

```diff
@@ -1,2 +1,3 @@
 export const API_URL =
-  'https://opentdb.com/api.php?amount=10&category=9&difficulty=easy&type=multiple';
+  'https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple';
+export const TOKEN = 'https://opentdb.com/api_token.php?command=request';
```

Comment:
Rename this constant.

It's not a token, but it's a URL that you can use to get a token.